README.md

```diff
@@ -16,7 +16,8 @@ Each member of the team is the owner of their tasks. They choose how to do their
 ## Responsibility
```

Comment:
Can we remove the block of `Responsibility` as it was integrated with `Ownership`?

Reply:
Done :)